net/wgpeerselector/files/usr/bin/wgpeerselector

```diff
@@ -111,9 +111,9 @@ function WGPeer:install_to_kernel()
 	end
 
 	local command = "wg set '" .. self.iface .. "' peer '" .. self.public_key .. "' " ..
-	                "allowed-ips '" .. table.concat(self.allowed_ips, ",") .. "' " ..
-	                "endpoint '" .. remote .. "' " ..
-	                "persistent-keepalive '" .. tostring(self.persistent_keepalive) .. "'"
+			"allowed-ips '" .. table.concat(self.allowed_ips, ",") .. "' " ..
+			"endpoint '" .. remote .. "' " ..
+			"persistent-keepalive '" .. tostring(self.persistent_keepalive) .. "'"
```

Comment:
Personally, I disagree with this change - I prefer tabs for indentation and spaces for alignment, so everything stays aligned regardless of tab size. Unfortunately, tooling support for such "smart tabs" is bad, and for C code, the Linux kernel coding style also disagrees and uses tabs for alignment...

Reply:
I found the option "SpacesAfterTabs" for the .editorconfig-checker.json, which gets rid of the warnings/errors from three files and allows using the Linux kernel style tabs + final spaces. Let me know, if that would work for you like this or if you would prefer another option.

Reply:
Would that setting allow using the current style of this file (with all alignment done using spaces) instead of the kernel style?

Reply:
I tried switching it in vim with `:%s/\t/        /gc`. But it seems that any time I add a new line it is again tab separated. Adding something like this does not help either, because the file is not ending with a .lua extension:

```
diff --git a/.editorconfig b/.editorconfig
index ce8045a5e06d..3638ffc4f82d 100644
--- a/.editorconfig
+++ b/.editorconfig
@@ -34,6 +34,8 @@ indent_style = space
 indent_size = 2
 
 [*.lua]
+indent_style = space
+indent_size = 8
 
 [{Makefile,*.mk}]
 indent_style = unset
```

Reply:
If that worked, it would also switch indentation to spaces. My whole point is that optimally, indentation should be tabs, alignment should be spaces, regardless of language.

Reply:
Resolved in v4, clarified in chat